lib/tasks/test_vmdb.rake

```diff
@@ -1,18 +1,35 @@
+require 'parallel_tests'
 require_relative "./evm_test_helper"
 
 if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
 namespace :test do
   namespace :vmdb do
     desc "Setup environment for vmdb specs"
-    task :setup => [:initialize, :verify_no_db_access_loading_rails_environment, :setup_db]
+    task :setup => [:initialize, :verify_no_db_access_loading_rails_environment] do
+      ParallelTests::CLI.new.run(["--type", "rspec"] + ["-e", "bin/rake evm:db:reset"])
+    end
 
     task :teardown
   end
 
-  desc "Run all core specs (excludes automation, migrations, replication, etc)"
-  RSpec::Core::RakeTask.new(:vmdb => [:initialize, "evm:compile_sti_loader"]) do |t|
+  desc "Run all vmdb specs; Use PARALLEL=1 to run in parallel."
```

Comment:
Yeah, I like 1 for the brevity in the description but you could almost think 1 means run on only one core.